paasta_tools/cli/cmds/logs.py

```diff
@@ -100,9 +103,37 @@ def add_subparser(subparsers):
         default=DEFAULT_SOA_DIR,
         help="define a different soa config directory",
     )
+
+    status_parser.add_argument(
+        '-a', '--ago', dest='time_ago',
+        help='How long ago to get logs from, uses friendly durations like 32m or 1w. Defaults to 30 minutes',
+    )
+    status_parser.add_argument(
+        '-t', '--time', dest='time',
```

Comment:
There is also precedent for the ``--from`` and ``--to`` style args elsewhere: 
- our main users (those at Yelp) are used to ``--min-date`` and ``--max-date`` to scribereader
- externally, ``docker logs`` has a ``--since`` arg (equivalent to our from)

How about ``--from`` and ``--to`` supporting the human readable style you're proposing? so you can do ``paasta logs --from "10 minutes ago" --to "5 minutes ago"`` with `--to` defaulting to now?